client/assets/stylesheets/_main.scss

```diff
@@ -46,12 +46,7 @@ body,
 }
 
 body {
-	background: var(--color-surface-backdrop);
-
-	&:has(.step-container-v2) {
-		background: #fdfdfd;
-	}
-
+	background: var( --color-main-background, #fcfcfc );
```

Comment:
Are the fallbacks really necessary? Looks redundant to me

Reply:
I just want to make it more robust since I’m not sure if there are any edge cases where the variable might be undefined.

Reply:
I'd echo this, since that particular variable gets used in a lot of spaces, even just in this PR, it'll add a lot of places to update should we ever want to. That reduces the utility of a variable a little bit, IMO.